tests/visualization_tests/test_param_importances.py

```diff
@@ -139,6 +140,24 @@ def test_get_param_importances_info_empty(
     )
 
 
+@pytest.mark.parametrize(
+    "specific_create_study,objective_names",
+    [(create_study, ["Foo"]), (_create_multiobjective_study, ["Foo", "Bar"])],
+)
+def test_get_param_importances_infos_custom_objective_names(
+    specific_create_study: Callable[[], Study], objective_names: list[str]
+) -> None:
+    study = specific_create_study()
+    study.set_metric_names(objective_names)
```

Comment:
How about testing the case of not setting the metric names?

Reply:
Added :+1: